### PR TITLE
Add instructions on how to start Elasticsearch

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/ElasticsearchQueryRunner.java [hunk 1]
```diff
@@ -38,6 +38,14 @@
 import static java.util.Locale.ENGLISH;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+/**
+ * Runs a Presto server with the Elasticsearch connector configured to talk
+ * to an external Elasticsearch instance.
+ *
+ * To start Elasticsearch:
+ *
+ * docker run -p 9200:9200 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.6.2
```
Comment (findepi) [inline presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/ElasticsearchQueryRunner.java:47]: This should be on `main()` method.

The class in general, does not expect ES to be running on a prticular port (the createQueryRunner method takes hostandport for this)

**Review by findepi — APPROVED** [review]
% comment placement

**Comment by findepi** [comment]
thanks!

**Review by electrum — APPROVED** [review]
We might want to make the query runner start and manage it automatically, like some of the others do.

**Comment by martint** [comment]
> We might want to make the query runner start and manage it automatically, like some of the others do.

See discussion in https://github.com/prestosql/presto/pull/3322#issuecomment-608512436

**Pull request merged**